core-bundle/src/EventListener/DataContainer/ValidateCustomRgxpListener.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\EventListener\DataContainer;
+
+use Contao\CoreBundle\ServiceAnnotation\Callback;
+use Symfony\Contracts\Translation\TranslatorInterface;
+
+/**
+ * @Callback(table="tl_form_field", target="fields.custom_rgxp.save")
+ */
+class ValidateCustomRgxpListener
+{
+    /**
+     * @var TranslatorInterface
+     */
+    private $translator;
+
+    public function __construct(TranslatorInterface $translator)
+    {
+        $this->translator = $translator;
+    }
+
+    public function __invoke($input)
+    {
+        if (false === @preg_match($input, '')) {
```

Comment:
how is this supposed to work? If the regex requires a value, it would fail?

Reply:
> If the regex requires a value, it would fail?

preg_match() returns 1 if the pattern matches given subject, 0 if it does not, or FALSE if an error occurred.

Reply:
How about adding a comment for that?
```suggestion
        // preg_match() returns false if the regular expression is invalid
        if (false === @preg_match($input, '')) {
```